From b49ef9b81d78874530e746d431273869a59876ee Mon Sep 17 00:00:00 2001
From: Charles Munson <jetwhiz@jetwhiz.com>
Date: Tue, 24 May 2016 13:24:19 +0200
Subject: Do not send logs to output file (easyloggingpp)


diff --git a/src/easylogging++.h b/src/easylogging++.h
--- a/src/easylogging++.h
+++ b/src/easylogging++.h
@@ -372,7 +372,10 @@ ELPP_INTERNAL_DEBUGGING_OUT_INFO << ELPP_INTERNAL_DEBUGGING_MSG(internalInfoStre
 #   include <sys/time.h>
 #elif ELPP_OS_WINDOWS
 #   include <direct.h>
+#   define WIN32_NO_STATUS
 #   include <windows.h>
+#   undef WIN32_NO_STATUS
+#   include <ntstatus.h>
 #  if defined(WIN32_LEAN_AND_MEAN)
 #      if defined(ELPP_WINSOCK2)
 #         include <winsock2.h>
